test/controllers/forms/QuestionnaireDiversityFormSpec.scala

```diff
@@ -29,7 +29,7 @@ class QuestionnaireDiversityFormSpec extends BaseSpec {
       actualData mustBe expectedData
     }
 
-    "fail when no gener" in new Fixture {
+    "fail when no gender" in new Fixture {
       assertFieldRequired("gender", "gender")
```

Comment:
it's not clear how this test works without checking the logic. Maybe this would help better:

```
assertFieldRequired(expectedError = "gender", "gender")
```